packages/framework-types/src/user-app.ts

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { BoosterApp } from '.'
 
 /**
- * `UserApp` represents the interface of the value when
- * we `require` the `dist/index.js` file of the user.
+ * `UserApp` represents the lambdas interface of a Booster app
+ * when it's dynamically imported from a different module.
  */
 export interface UserApp {
-  Booster: BoosterApp
```

Comment:
Still don't quite understand why this was removed. When we load the project, the field `Booster` is exported and it is needed to be used in the core and in the local provider.

Reply:
This field wasn't used at all, all usages of the `UserApp` interface only accessed the lambdas.